gen/vsup_ifis

```diff
@@ -133,15 +156,18 @@ foreach my $rData ($data->getChildElements) {
 			$users->{$key}->{'ADR_TYPE'} = '';
 		}
 
-		if ($uAttributes{$A_BANK_ACCOUNT_DC2}) {
-			$users->{$key}->{'BANK_ACCOUNT'} = $uAttributes{$A_BANK_ACCOUNT_DC2};
-		} elsif ($uAttributes{$A_BANK_ACCOUNT_KOS}) {
-			$users->{$key}->{'BANK_ACCOUNT'} = $uAttributes{$A_BANK_ACCOUNT_KOS};
+		my $bankAccountDC2 = $data->getUserAttributeValue(member => $memberId, attrName => $A_BANK_ACCOUNT_DC2);
+		my $bankAccountKOS = $data->getUserAttributeValue(member => $memberId, attrName => $A_BANK_ACCOUNT_KOS);
+
+		if ($bankAccountDC2) {
+			$users->{$key}->{'BANK_ACCOUNT'} = $bankAccountDC2;
+		} elsif ($bankAccountKOS) {
+			$users->{$key}->{'BANK_ACCOUNT'} = $bankAccountKOS;
 		} else {
 			$users->{$key}->{'BANK_ACCOUNT'} = '';
 		}
 
-		$users->{$key}->{$A_USER_ID} = $uAttributes{$A_USER_ID};
+		$users->{$key}->{$A_USER_ID} = $data->getUserAttributeValue(member => $memberId, attrName => $A_USER_ID);
```

Comment:
We can replace this with ``getUserIdForMember(member => $memberId)`` and remove ```user:core:id``` attribute from the required attributes of the service.

Reply:
Fixed